src/main/java/com/github/ogomezso/kafka/connect/soap/source/SoapSourceConnectorConfig.java

```diff
@@ -0,0 +1,81 @@
+/**
+ *  Copyright © 2021 Oscar Gómez (ogomezso0@gmail.com)
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.github.ogomezso.kafka.connect.soap.source;
+
+import java.io.File;
+import java.util.Map;
+
+import org.apache.kafka.common.config.AbstractConfig;
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.common.config.ConfigDef.Importance;
+import org.apache.kafka.common.config.ConfigDef.Type;
+
+import com.github.jcustenborder.kafka.connect.utils.config.ConfigUtils;
+
+public class SoapSourceConnectorConfig extends AbstractConfig {
+
+  public static final String ENDPOINT_URL = "endpointUrl";
```

Comment:
consider to expose a CONNECTION_TIMEOUT configuration.

Reply:
now in connector config with default value